features/init.feature

```diff
@@ -101,29 +101,29 @@ Scenario: When creating a bitbucket backed project
   And I should be checked out to a feature branch named "initialize-delivery-pipeline"
   And a change should be created for branch "initialize-delivery-pipeline"
 
-Scenario: When creating a github backed project
-  When a user creates a github backed project
-  Then a github project is created in delivery
+Scenario: When creating a GitHub backed project
+  When a user creates a GitHub backed project
```

Comment:
You are changing the cucumber steps, you need to also modify these changes inside `features/support/steps.rb`